Source/WebCore/layout/formattingContexts/inline/InlineItem.h

```diff
@@ -70,6 +70,7 @@ class InlineItem {
 
 private:
     friend class InlineItemsBuilder;
+    friend class TextOnlySimpleLineBuilder;
```

Comment:
This looks like layering violation...and I can't figure out why it is needed at all.

Reply:
ops, this is some left over from a first approach. My first idea was to change the width of the inline text item itself for trimming, but I've realized I could just alter the builder's logical width